<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  


























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Technologies for mitochondrial genomics and lineage tracing | Lareau Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Technologies for mitochondrial genomics and lineage tracing">
<meta name="description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">

<meta property="og:title" content="Technologies for mitochondrial genomics and lineage tracing">
<meta property="og:site_title" content="Lareau Lab">
<meta property="og:description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">
<meta property="og:url" content="">
<meta property="og:image" content="">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Technologies for mitochondrial genomics and lineage tracing">
<meta property="twitter:description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">
<meta property="twitter:url" content="">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Technologies for mitochondrial genomics and lineage tracing",
    "description": "The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.",
    "headline": "Technologies for mitochondrial genomics and lineage tracing",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": ""
  }
</script>

<link rel="alternate" type="application/rss+xml" href="/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Soleil:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Tisa:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet';">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/scistories.jpg')" data-dark="false">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewbox="-1 -1 302 63">
  <defs>
    <style>
      .cls-1 {
        fill: #3963af;
      }

      .cls-2 {
        fill: #00aee6;
      }

      .cls-3 {
        stroke: #00aee6;
        fill: none;
        stroke-miterlimit: 5;
        stroke-width: 4px;
        stroke-dasharray: 500;
        stroke-dashoffset: 500;
        animation: dash 5s linear forwards;
      }
      .cls-4 {
        stroke: #3963af;
        stroke-width: 4px;
        fill: none;
        stroke-miterlimit: 5;
        stroke-width: 4px;
        stroke-dasharray: 500;
        stroke-dashoffset: 500;
        animation: dash 5s linear forwards;
      }
            @keyframes dash {
  to {
    stroke-dashoffset: 0;
  }
  }
    </style>
  </defs>
  <polygon class="cls-1" points="58.6 45.8 58.6 13.5 66.9 13.5 66.9 38.6 76.4 38.6 76.4 45.8 58.6 45.8"></polygon>
  <path class="cls-1" d="M104.5,48.8V46.2h-.1c-1,2.1-3.9,3.4-7.1,3.4-7.2,0-12.1-5.6-12.1-12.9s5.2-13,12.1-13a8.2,8.2,0,0,1,7.1,3.4h.1V24.6h7.3V48.8Zm0-12.1a6.1,6.1,0,0,0-6.1-6.2,6.1,6.1,0,0,0-6,6.2,6.1,6.1,0,0,0,6,6.2,6,6,0,0,0,6.1-6.2" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M119,48.8V24.6h6.9v2.5h0c1.3-2.5,3.3-3.4,6.1-3.4V31c-4.3.1-5.7,1.7-5.7,4.4V48.8Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M142.8,38.9a5.5,5.5,0,0,0,5.7,4.6,5.3,5.3,0,0,0,4.4-2.3h7.4c-2.3,5.3-6.7,8.4-11.8,8.4a13,13,0,0,1-.2-25.9c7.3,0,12.8,5.7,12.8,13.3a6.2,6.2,0,0,1-.2,1.9ZM153.9,34a5.5,5.5,0,0,0-5.6-4.2,5.3,5.3,0,0,0-5.4,4.2Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M210.5,48.8V46.2h-.1c-1,2.3-2.9,3.4-6.2,3.4-5.1,0-9.4-3.1-9.4-10v-15h7.3V38.1c0,3.2,1.1,4.7,3.9,4.7s4.1-1.8,4.1-4.8V24.6h7.3V48.8Z" transform="translate(-4.9 -3)"></path>
  <polygon class="cls-2" points="220.5 45.8 220.5 13.5 223.8 13.5 223.8 42.9 237.1 42.9 237.1 45.8 220.5 45.8"></polygon>
  <path class="cls-2" d="M267,48.8V44.6h-.1a10.9,10.9,0,0,1-9.3,4.8,12.6,12.6,0,1,1,0-25.1,10.7,10.7,0,0,1,9.3,4.8h.1V24.9h3.1V48.8Zm.2-11.9a9.7,9.7,0,1,0-19.4,0,9.7,9.7,0,1,0,19.4,0" transform="translate(-4.9 -3)"></path>
  <path class="cls-2" d="M278,48.8V16.5h3V29.1h.1a11.5,11.5,0,0,1,9.3-4.8,12.6,12.6,0,1,1,0,25.1,11.3,11.3,0,0,1-9.3-4.8H281v4.2ZM290.4,27a9.8,9.8,0,0,0-9.5,9.9,9.6,9.6,0,0,0,9.5,9.7,9.5,9.5,0,0,0,9.8-9.7,9.6,9.6,0,0,0-9.8-9.9" transform="translate(-4.9 -3)"></path>
  <path class="cls-3" d="M38.5,45.2a4.8,4.8,0,0,1-4.8-4.7V9.7a4.8,4.8,0,1,0-9.5,0V31.9A4.7,4.7,0,0,0,29,36.7H48.4a4.3,4.3,0,1,1,0,8.5Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-4" d="M21.1,61.2a4.7,4.7,0,0,1-4.7-4.8V25.7a4.8,4.8,0,1,0-9.6,0V47.9a4.8,4.8,0,0,0,4.8,4.8H31.1a4.3,4.3,0,0,1,0,8.5Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M183.4,49V46.4h-.1c-1.1,2.1-4,3.4-7.2,3.4-7.1,0-12.1-5.6-12.1-13s5.2-12.9,12.1-12.9c3,0,5.8,1.2,7.2,3.4h.1V24.7h7.2V49Zm0-12.2a6.1,6.1,0,0,0-6.2-6.1,6.2,6.2,0,0,0,.1,12.3,6.1,6.1,0,0,0,6.1-6.2" transform="translate(-4.9 -3)"></path>
</svg>

        
      </span>
    
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/team/" data-tooltip="Who we are">
          Team
        </a>
      
    
      
        <a href="/news/" data-tooltip="Updates from the lab">
          News
        </a>
      
    
      
        <a href="/research/" data-tooltip="Scientific focuses">
          Research
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Lab papers">
          Publications
        </a>
      
    
      
        <a href="/reproducibility/" data-tooltip="Code &amp; protocols for our work">
          Reproducibility
        </a>
      
    
      
        <a href="/outreach/" data-tooltip="Get involved or join!">
          Outreach
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Get in touch!">
          Contact
        </a>
      
    
  </nav>
</header>
    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - wrap each table in div to allow for scrolling
  - filter out blank sections
-->








  
  
  

  <section class="background" data-size="page">
    <h1 id="methods-for-mtdna-tracing">
<i class="icon fa-solid fa-dna"></i>Methods for mtDNA tracing</h1>

<p>Our lab has been active in the development of new technologies to enable
lineage tracing in humans via mitochondrial DNA mutations. Most of these works have been 
collaborative efforts with the <a href="https://www.mdc-berlin.de/ludwig">Leif Ludwig</a> Lab
while we were trainees with Aviv Regev and Vijay Sankaran. See a summary of these
major technologies, software, biological results, and informal guidelines below.</p>

<h2 id="interested-in-mitochondrial-tracing">Interested in mitochondrial tracing?</h2>

<p>A good review on these principles, technologies, and applications
is <a href="https://www.nature.com/articles/s41588-024-01794-8">available online as a review article</a>.
Feel free to <a href="mailto:lareauc@mskcc.org">reach out to Caleb</a>, who can either answer your questions
or refer you to the right person who can help!</p>

<p>At a high level, we can detect somatic mitochondrial mutations in either mitochondrial DNA (mtDNA) or 
mtRNA. These methods that we’ve been involved in developing are summarized here:</p>

<h1 id="genotyping-via-mtdna">Genotyping via mtDNA</h1>

<h2 id="mitochondrial-single-cell-atac-seq">Mitochondrial single-cell ATAC-seq</h2>

<p>The mitochondrial single-cell ATAC-seq (mtscATAC-seq) protocol is an adaptation of 
commercial single-cell ATAC-seq workflows. At a basic level, by fixing and permeabilizing
cells (versus just nuclei), Tn5 also tagments the more accessible mitochondrial genome.
Accessible chromatin profiling can thereby be combined with single-cell whole mitochondrial genome sequencing.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/mtscatac-overview-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<h2 id="asap-seq-and-dogma-seq">ASAP-seq and DOGMA-seq</h2>

<p><a href="https://www.nature.com/articles/s41587-021-00927-2">ATAC with select antigen profiling by sequencing (ASAP-seq)</a> (with Eleni Mimitou and Peter Smibert);</p>

<p>ASAP-seq (ATAC with Select Antigen Profiling by Sequencing) enables a multimodal readout that simultaneously profiles accessible chromatin, protein levels, and optionally mitochondrial DNA for cellular clonality by repurposing the existing antibody-oligonucleotide conjugates (Figure 4A) [16]. As the bead oligo capture sequences on the scATAC-seq kit are complementary to the Nextra Read 1 sequence attached to DNA fragments after Tn5 transposition, the poly-A CITE-seq reagents are not immediately compatible. Thus, the critical innovation in ASAP-seq was a bridge oligo complementary to the poly-A antibody-oligo sequence on one end and the Read 1 sequence on the other. This development allows existing CITE-seq reagents to be used directly in ASAP-seq without requiring a new set of antibodies to be conjugated for capture with the ATAC-seq kit.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/asap-seq-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>The 10x Genomics Multiome kit has further facilitated the integration of transcriptional profiles,
yielding <a href="https://www.nature.com/articles/s41587-021-00927-2">DOGMA-seq</a>; 
this enables the single-cell capture of up to four data modalities, including transcriptome,
accessible chromatin, surface markers and mtDNA mutations.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/dogma-seq-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<h2 id="phage-atac">PHAGE-ATAC</h2>
<p><a href="https://www.nature.com/articles/s41587-021-01065-5">PHAGE-ATAC</a> (with Evgenij Fiskin),
which in addition facilitate surface-marker profiling. In our experience, these approaches both work well 
and right-out-of-the-box if you have experience working with antibodies and/or phages.</p>

<p>PHAGE-ATAC utilizes an M13 bacteriophage system that genetically encoded antibody binders
attached to a PAC-tag, obviating the need for recombinantly expressed antibodies or covalent conjugations.
The M13 phagemid system uses nanobodies with known specificity to detect surface antigens
and quantify binding in droplets via the barcoding of the CDR3 hypervariable region that directly encodes the protein binder.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/phage-atac-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>As phages are effectively a renewable resource after infection in bacteria, these reagents are ideal for large-scale experiments.
Keep an eye on this assay as additional resources outlining nanobodies are developed.</p>

<p>Unless you have a specific reason to want the transcriptome (then use DOGMA-seq or MAESTER, below) or 
the surface proteome (then use ASAP-seq due to ease of commercially available reagents),
we recommend mtscATAC-seq due to the ease and cost-efficiency of the workflow.</p>

<p>Another extension of the mtscATAC-seq assay (though unrelated to our group) for the genotyping of 
targeted loci with single-cell chromatin accessibility <a href="https://www.nature.com/articles/s41586-024-07388-y">(GoT-ChA)</a>
utilizes targeted primers to genotype nuclear variants in addition to optional mtDNA sequencing for high-resolution clonal analyses.</p>
  </section>

  
  
  

  <section class="background" data-size="2" data-dark="false" style="--image: url('/images/scistories-clear-cut.png')">
    <!--
  background: images/scistories-clear-cut.png;
  dark: false;
  size: 2;
-->
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<h1 id="genotyping-via-mtrna">Genotyping via mtRNA</h1>

<h2 id="maester">MAESTER</h2>
<p>As &gt;90% of mtDNA is transcribed, full-length RNA-seq techniques (e.g., Smart-seq) that capture
the entire sequence of a transcript are particularly attractive to capture genetic variants.
As relatively few groups use Smart-seq nowadays, approaches that are compatible with 
the more high-throughput and more popular 3′- or 5′-based scRNA-seq approaches were necessary. 
However, it’s important to note that these approaches sequence 
only part of the transcript and thus tend to suffer from limited coverage of the mitochondrial
transcriptome for confident variant detection.</p>

<p>Enter <a href="https://pubmed.ncbi.nlm.nih.gov/35210612/">(MAESTER)</a>.
MAESTER can enrich mitochondrial mutations from existing cDNA libraries,
massively improving the suitability of these data for detecting and analyzing mtDNA
variants from high-throughput scRNA-seq data. 
MAESTER was the brain child of 
<a href="https://tymillerlab.org/">Tyler Miller</a> and <a href="https://vangalenlab.bwh.harvard.edu/">Peter van Galen</a>, 
and we were happy to help out in finalizing its development.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/maester-overview-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>In general, if you are regular user of 10x scRNA-seq, MAESTER may be a great entry-level 
assay for looking at mitochondrial genomes in your cells
However, there are a few noteworthy limitations of genotyping mitochondrial mutations from RNA alone.
First, note a high degree of inherent transcriptional error of the mitochondrial RNA polymerase,
which we have observed will lead to false-positive heteroplasmy detection. 
Additionally, not all mtDNA molecules may be equally transcribed or captured, including 
the 22 mitochondrial tRNAs are not reverse transcribed by poly(A)-based priming. 
We advise to download public data related to the assay that you plan to use
and compute the per-singe-cell coverage to get an estimate of what your data may look like.</p>

<p>One of the best applications of MAESTER resolved the 
<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9900873/">malignant evolution in Barrett’s esophagus</a>
(from <a href="https://www.sydshafferlab.com/">Syd Shaffer’s Lab</a>) providing a unified view of the cell
states and dynamics along a disease trajectory.</p>
  </section>

  
  
  

  <section class="background" data-size="2" data-dark="false" style="--image: url('/images/scistories-clear-cut.png')">
    <!--
  background: images/scistories-clear-cut.png;
  dark: false;
  size: 2;
-->
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<h1 id="purifying-selection-in-congenital-mitochondrial-disease">Purifying selection in congenital mitochondrial disease</h1>

<p>While most single-cell multi-omic efforts so far have used mtDNA genotyping for purposes of clonal tracing,
the obvious utility of these technologies for studies of mitochondrial genetics has only
become evident more recently in the context of mitochondriopathies.
Clinically, these heterogeneous diseases are well recognized, but a detailed molecular understanding of 
how alterations in mitochondrial genetic integrity elicit changes in individual cells.</p>

<p>We initially applied mtscATAC-seq-based analysis of peripheral blood mononuclear cells
from adult patients with mitochondrial encephalomyopathy, lactic acidosis and stroke-like episodes (MELAS),
who carry the pathogenic m.3243G&gt;A mtDNA variant. In collaboration with Vamsi Mootha and Melissa Walker,
We found an almost complete absence of the mutation, specifically in T cells but not in other immune cells.</p>

<p>With Suneet Agarwal, we further applied mtscATAC-seq, ASAP-seq, and DOGMA-seq</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/pearson-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>This work showed a strong purifying selection of pathogenic mtDNA in cell states that probably reflect
specific metabolic requirements, in particular toward attaining and/or also potentially 
maintaining the mucosal-associated invariant T cell (MAIT) and CD8+ effector/memory T cell states.</p>

<p>In short, if you want to learn more about how functional mtDNA variants impact cells, the above suite of 
technologies has worked well in our hands!</p>
  </section>

  
  
  

  <section class="background" data-size="2" data-dark="false" style="--image: url('/images/scistories-clear-cut.png')">
    <!--
  background: images/scistories-clear-cut.png;
  dark: false;
  size: 2;
-->
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  background: ;
  dark: ;
  size: ;
-->

<h1 id="software-that-enables-mitochondrial-mutaiton-analyses">Software that enables mitochondrial mutaiton analyses</h1>

<p>As part of these tools, our lab has been the primary developers of genotyping software for processing sequencing reads 
from the protocols and identifying high-confidence single-cell heteroplasmy.</p>

<ul>
  <li>For analyzing mtDNA (from any ATAC data), we recommend the <a href="https://github.com/caleblareau/mgatk">mgatk package</a>.</li>
  <li>To call mtDNA deletions and quantify single-cell heteroplasmy, we recommend the <a href="https://github.com/caleblareau/mgatk">mgatk-del utility</a> part of mgatk. A detailed summary of the <a href="https://github.com/caleblareau/mgatk/wiki/Large-deletion-calling-and-heteroplasmy-estimation">toolkit is here</a>
</li>
  <li>For analyzing MAESTER/mRNA data, we recommend the <a href="https://github.com/caleblareau/maegatk">maegatk workflow</a>.</li>
</ul>

<p>The major differences between the software tools <a href="https://github.com/caleblareau/maegatk/wiki/FAQ">are described here</a>.</p>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/scistories.jpg')" data-dark="false" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->
  <div>
    
  </div>
  
  <div>
  <b>Data-driven translational genomics</b><br>
   Lareau Lab
    | <a href="https://www.mskcc.org/research/ski/programs/computational-biology">csBio</a>
    | MSKCC
  </div>
</footer>

  </body>
</html>
