<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  


























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Identifying multiple beads per droplet in ATAC-seq data | Lareau Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Identifying multiple beads per droplet in ATAC-seq data">
<meta name="description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">

<meta property="og:title" content="Identifying multiple beads per droplet in ATAC-seq data">
<meta property="og:site_title" content="Lareau Lab">
<meta property="og:description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">
<meta property="og:url" content="">
<meta property="og:image" content="">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Identifying multiple beads per droplet in ATAC-seq data">
<meta property="twitter:description" content="The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.">
<meta property="twitter:url" content="">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Identifying multiple beads per droplet in ATAC-seq data",
    "description": "The Lareau Laboratory for Data-driven Translational Genomics is in the csBio Program at Memorial Sloan Kettering Cancer Center, New York, NY.",
    "headline": "Identifying multiple beads per droplet in ATAC-seq data",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": ""
  }
</script>

<link rel="alternate" type="application/rss+xml" href="https://clareaulab.com/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Soleil:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Tisa:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="preload" as="style" onload="this.onload = null; this.rel = 'stylesheet';">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/scistories.jpg')" data-dark="false">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewbox="-1 -1 302 63">
  <defs>
    <style>
      .cls-1 {
        fill: #3963af;
      }

      .cls-2 {
        fill: #00aee6;
      }

      .cls-3 {
        stroke: #00aee6;
        fill: none;
        stroke-miterlimit: 5;
        stroke-width: 4px;
        stroke-dasharray: 500;
        stroke-dashoffset: 500;
        animation: dash 5s linear forwards;
      }
      .cls-4 {
        stroke: #3963af;
        stroke-width: 4px;
        fill: none;
        stroke-miterlimit: 5;
        stroke-width: 4px;
        stroke-dasharray: 500;
        stroke-dashoffset: 500;
        animation: dash 5s linear forwards;
      }
            @keyframes dash {
  to {
    stroke-dashoffset: 0;
  }
  }
    </style>
  </defs>
  <polygon class="cls-1" points="58.6 45.8 58.6 13.5 66.9 13.5 66.9 38.6 76.4 38.6 76.4 45.8 58.6 45.8"></polygon>
  <path class="cls-1" d="M104.5,48.8V46.2h-.1c-1,2.1-3.9,3.4-7.1,3.4-7.2,0-12.1-5.6-12.1-12.9s5.2-13,12.1-13a8.2,8.2,0,0,1,7.1,3.4h.1V24.6h7.3V48.8Zm0-12.1a6.1,6.1,0,0,0-6.1-6.2,6.1,6.1,0,0,0-6,6.2,6.1,6.1,0,0,0,6,6.2,6,6,0,0,0,6.1-6.2" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M119,48.8V24.6h6.9v2.5h0c1.3-2.5,3.3-3.4,6.1-3.4V31c-4.3.1-5.7,1.7-5.7,4.4V48.8Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M142.8,38.9a5.5,5.5,0,0,0,5.7,4.6,5.3,5.3,0,0,0,4.4-2.3h7.4c-2.3,5.3-6.7,8.4-11.8,8.4a13,13,0,0,1-.2-25.9c7.3,0,12.8,5.7,12.8,13.3a6.2,6.2,0,0,1-.2,1.9ZM153.9,34a5.5,5.5,0,0,0-5.6-4.2,5.3,5.3,0,0,0-5.4,4.2Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M210.5,48.8V46.2h-.1c-1,2.3-2.9,3.4-6.2,3.4-5.1,0-9.4-3.1-9.4-10v-15h7.3V38.1c0,3.2,1.1,4.7,3.9,4.7s4.1-1.8,4.1-4.8V24.6h7.3V48.8Z" transform="translate(-4.9 -3)"></path>
  <polygon class="cls-2" points="220.5 45.8 220.5 13.5 223.8 13.5 223.8 42.9 237.1 42.9 237.1 45.8 220.5 45.8"></polygon>
  <path class="cls-2" d="M267,48.8V44.6h-.1a10.9,10.9,0,0,1-9.3,4.8,12.6,12.6,0,1,1,0-25.1,10.7,10.7,0,0,1,9.3,4.8h.1V24.9h3.1V48.8Zm.2-11.9a9.7,9.7,0,1,0-19.4,0,9.7,9.7,0,1,0,19.4,0" transform="translate(-4.9 -3)"></path>
  <path class="cls-2" d="M278,48.8V16.5h3V29.1h.1a11.5,11.5,0,0,1,9.3-4.8,12.6,12.6,0,1,1,0,25.1,11.3,11.3,0,0,1-9.3-4.8H281v4.2ZM290.4,27a9.8,9.8,0,0,0-9.5,9.9,9.6,9.6,0,0,0,9.5,9.7,9.5,9.5,0,0,0,9.8-9.7,9.6,9.6,0,0,0-9.8-9.9" transform="translate(-4.9 -3)"></path>
  <path class="cls-3" d="M38.5,45.2a4.8,4.8,0,0,1-4.8-4.7V9.7a4.8,4.8,0,1,0-9.5,0V31.9A4.7,4.7,0,0,0,29,36.7H48.4a4.3,4.3,0,1,1,0,8.5Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-4" d="M21.1,61.2a4.7,4.7,0,0,1-4.7-4.8V25.7a4.8,4.8,0,1,0-9.6,0V47.9a4.8,4.8,0,0,0,4.8,4.8H31.1a4.3,4.3,0,0,1,0,8.5Z" transform="translate(-4.9 -3)"></path>
  <path class="cls-1" d="M183.4,49V46.4h-.1c-1.1,2.1-4,3.4-7.2,3.4-7.1,0-12.1-5.6-12.1-13s5.2-12.9,12.1-12.9c3,0,5.8,1.2,7.2,3.4h.1V24.7h7.2V49Zm0-12.2a6.1,6.1,0,0,0-6.2-6.1,6.2,6.2,0,0,0,.1,12.3,6.1,6.1,0,0,0,6.1-6.2" transform="translate(-4.9 -3)"></path>
</svg>

        
      </span>
    
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/team/" data-tooltip="Who we are">
          Team
        </a>
      
    
      
        <a href="/news/" data-tooltip="Updates from the lab">
          News
        </a>
      
    
      
        <a href="/research/" data-tooltip="Scientific focuses">
          Research
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Lab papers">
          Publications
        </a>
      
    
      
        <a href="/reproducibility/" data-tooltip="Code &amp; protocols for our work">
          Reproducibility
        </a>
      
    
      
        <a href="/outreach/" data-tooltip="Get involved or join!">
          Outreach
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Get in touch!">
          Contact
        </a>
      
    
  </nav>
</header>
    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - wrap each table in div to allow for scrolling
  - filter out blank sections
-->








  
  
  

  <section class="background" data-size="page">
    <h1 id="what-are-barcode-multiplets">
<i class="icon fa-solid fa-droplet"></i>What are barcode multiplets?</h1>

<p>A common / foundational assumption in single-cell genomics analyses is that a single oligonucleotide barcode
represents the nucleic acid of an individual cell. However, there are scenarios where an individual single cell 
can have its contents barcoded by multiple distinct oligonucleotides, termed barcode multiplets. 
These scenarios include i) a single droplet containing multiple beads or ii) multiple oligonucleotides
present on the same individual bead. Every single-cell assay has barcode multiplets to varying degrees.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/barcode-multiplets-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<h2 id="barcode-multiplets-used-to-develop-dscatac-seq">Barcode multiplets used to develop dscATAC-seq</h2>

<p>Prior work has shown that if you adjust the size/composition of the beads,
one can control the number of beads loaded into each droplet. 
10x Genomics uses these soft hydrogel beads to ensure that you get 1 bead per droplet.</p>

<p>For small beads, including those used in Drop-seq and other home-brew settings,
this is hard to achieve, including in the commercialized BioRad ddSeq platform. 
This is an obstacle because you split data</p>

<p>To get around this, we superloaded beads to reduce the number of droplets with zero beads.
In the process, we greatly increase the number of droplets with 2+ bead
Fortunately, due to droplet-PCR, beads in the same droplet will share the same transposition 
positions of genomic fragments, enabling a statistic that can pair barcode multiplets. We call this tool
<a href="https://github.com/caleblareau/bap">bap</a> that can detect barcode multiplets efficently.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/biorad-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>Using <code class="language-plaintext highlighter-rouge">bap</code>, the <a href="https://www.bio-rad.com/en-us/product/ddseq-single-cell-atac-seq-kit-omnition-analysis-software?ID=PEXSR1MC1ORV">BioRad ddSEQ ATAC</a>
kit could become a reality. By merging barcode multiplets, the approach had very low cell dropout / high cell recovery.</p>

<h2 id="accuracy-of-bap">Accuracy of bap</h2>

<p>Implicit in all of this is the requirement of bap to be very accurate– you wanted to merge everything
that should be merged and miss nothing that shouldn’t. To test this, we designed an experiment with
diverse exogenous oligonucleotides to establish a set of true beads that were co-contained in droplets
alongside capture of a cell and then ran <code class="language-plaintext highlighter-rouge">bap</code> to recover the pairs of barcodes.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/bap-performance.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>The results were clear: at any bead concentration, <code class="language-plaintext highlighter-rouge">bap</code> had an AUROC/AUPRC of 0.999+, 
meaning we could trivially classify barcode multiplets from single-cell data.</p>

<h2 id="application-to-10x-genomics-scatac-data">Application to 10x Genomics scATAC data</h2>

<p>When applying bap to scATAC-seq datasets released from 10x Genomics, instances of 9+ bead barcodes
were annotated by bap of belonging to the same droplet (all barcodes annotated as ‘cells’
having a sequence in the whitelist and read abundances above the knee threshold).
These barcodes were enriched in the same biological cluster, supportive of the idea that
they may be ‘replicates’ of the same cell.</p>

<p><a href="https://github.com/caleblareau/bap/wiki/10X-scATAC">fully supports processing 10x Genomics data</a>.</p>

<figure class="figure">
  <a class="figure-image" aria-label="figure link">
    <img src="/images/researchpngs/10x-bm-wide.png" style="
        width: auto;
        max-height: unset;
      " alt="figure image" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
  </a>
  
</figure>

<p>Taking a closer look at the underlying barcode sequence,
a common 7 nucleotide variable region followed by a 9 base constant in these barcodes, 
consistent with the idea that multiple ‘whitelisted’ barcode sequences were installed on an individual bead during synthesis.</p>

<p>As far as we know, this issue was prevalent in the early ATAC beads that were synthesized 
by 10x but have been since resolved in the ATAC and other kits. Further, 
a software implementation (that gives nearly identical results to bap)
realeased from 10x Genomics was incorporated into CellRanger-ATAC v1.2. 
This functionality similarly analyzes shared transposition events between pairs of barcode
sequences to identify multiplets. Whereas bap merges different bead barcode sequences
into a single cell barcode, the implementation in CellRanger-ATAC simply discards
multiplet barcodes after retaining the one bead barcode with the highest number of fragments.</p>

<p>In short, as long as you are using v1.2+ of CellRanger-ATAC, you shouldn’t have to worry about barcode multiplets.</p>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/scistories.jpg')" data-dark="false" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->
  <div>
    
  </div>
  
  <div>
  <b>Data-driven translational genomics</b><br>
   Lareau Lab
    | <a href="https://www.mskcc.org/research/ski/programs/computational-biology">csBio</a>
    | MSKCC
  </div>
</footer>

  </body>
</html>
